docs/docs/advanced/deploy-eliza-to-tee-with-oasis-rofl.md

```diff
@@ -0,0 +1,257 @@
+    image: ghcr.io/elizaos/eliza:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    platform: linux/amd64
+    environment:
+      - CHARACTERS=${CHARACTERS}
+      - SERVER_PORT=${SERVER_PORT}
+```
+
+---
+
+### Secret Management
+
+The secret management system handles secure storage and access to sensitive data. It:
+
+- Encrypts environment variables and secrets
+- Provides secure access to sensitive data
+- Supports both development and production environments
+
+Key features:
+- Secure secret storage
+- Environment variable encryption
+- Access control
+- Audit logging
+
+Here's a comprehensive list of secrets you might want to set for your Eliza agent:
+
+```bash
+# Basic Configuration
+# Set the character file path (you can use any character file of your choice)
+echo -n "characters/staff-engineer.character.json" | oasis rofl secret set CHARACTERS -
+echo -n "3000" | oasis rofl secret set SERVER_PORT -
+echo -n "5173" | oasis rofl secret set UI_PORT -
+echo -n "" | oasis rofl secret set UI_ALLOWED_HOSTS -
+echo -n "" | oasis rofl secret set UI_SERVER_URL -
+echo -n "" | oasis rofl secret set REMOTE_CHARACTER_URLS -
+echo -n "false" | oasis rofl secret set USE_CHARACTER_STORAGE -
+
+# Logging Configuration
+echo -n "log" | oasis rofl secret set DEFAULT_LOG_LEVEL -
+echo -n "false" | oasis rofl secret set LOG_JSON_FORMAT -
+echo -n "false" | oasis rofl secret set INSTRUMENTATION_ENABLED -
+echo -n "" | oasis rofl secret set EXPRESS_MAX_PAYLOAD -
+
+# OpenAI Configuration
+echo -n "sk-proj-XXXX" | oasis rofl secret set OPENAI_API_KEY -
+echo -n "" | oasis rofl secret set OPENAI_API_URL -
+echo -n "" | oasis rofl secret set SMALL_OPENAI_MODEL -
+echo -n "" | oasis rofl secret set MEDIUM_OPENAI_MODEL -
+echo -n "" | oasis rofl secret set LARGE_OPENAI_MODEL -
+echo -n "" | oasis rofl secret set EMBEDDING_OPENAI_MODEL -
+echo -n "" | oasis rofl secret set IMAGE_OPENAI_MODEL -
+echo -n "false" | oasis rofl secret set USE_OPENAI_EMBEDDING -
+
+# GitHub Integration
+echo -n "true" | oasis rofl secret set GITHUB_CLIENT_DISABLED -
+echo -n "true" | oasis rofl secret set GITHUB_PLUGIN_ENABLED -
+echo -n "ghp_XXXX" | oasis rofl secret set GITHUB_API_TOKEN -
+echo -n "5000" | oasis rofl secret set GITHUB_USER_CHECK_INTERVAL_MS -
+echo -n "5000" | oasis rofl secret set GITHUB_INFO_DISCOVERY_INTERVAL_MS -
+echo -n "20000" | oasis rofl secret set GITHUB_OODA_INTERVAL_MS -
+echo -n "10" | oasis rofl secret set GITHUB_ISSUES_LIMIT -
+echo -n "10" | oasis rofl secret set GITHUB_PULL_REQUESTS_LIMIT -
+
+# Database Configuration
+echo -n "postgresql://user:password@localhost:5432/eliza" | oasis rofl secret set POSTGRES_URL -
+```
+
+## Tutorial
+
+---
+
+### Prerequisites
+
+Before getting started with Eliza in TEE, ensure you have:
+
+- [Oasis CLI](https://github.com/oasisprotocol/cli/releases) installed
+- Docker Desktop or Orbstack (Orbstack is recommended)
+- Testnet tokens from the [Oasis Faucet](https://faucet.testnet.oasis.dev/)
```

Comment:
This is redundant here IMO, because you probably don't have a keypair yet which is described below.

```suggestion
```